ontology/yaml/resources/METERS/entity_types/EM.yaml

```diff
@@ -97,3 +97,11 @@ EM_US_SVL_TC1_1:
   uses:
   - average_line_current_sensor
   - average_inter_line_voltage_sensor
+#AU-SYD-ODI meter types
+EM_SE_PM5100: 
+  description: "SE PM5100 Meter"
+  uses:
+  - line_neutral_current_sensor
```

Comment:
explain this one: is it really voltage, or is this truly a differential current? if its a differential current, should be **line_neutral_differential_current_sensor**

Reply:
It really is a sensor monitoring current on the neutral line. I've added comments alongside.

Reply:
ohh i understand -- it isnt line to neutral, its the neutral line. change it to **neutral_line_current_sensor** and i think we are good.

Reply:
Ok, done.